Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/PersonaAlias.java

```diff
@@ -103,7 +114,35 @@ static PersonaAlias addPersonaAlias(Persona persona, String alias, String justif
                 + ")";
 
         CentralRepository.getInstance().executeInsertSQL(insertClause);
-        return new PersonaAlias(persona.getId(), alias, justification, confidence, timeStampMillis, examiner);
+        
+        String queryClause = SELECT_QUERY_BASE
+                + "WHERE pa.persona_id = " + persona.getId()
+                + " AND pa.alias = \"" + alias + "\""
+                + " AND pa.date_added = " + timeStampMillis
+                + " AND pa.examiner_id = " + examiner.getId();
+        
+        PersonaAliasesQueryCallback queryCallback = new PersonaAliasesQueryCallback();
+        CentralRepository.getInstance().executeSelectSQL(queryClause, queryCallback);
+        
+        Collection<PersonaAlias> aliases = queryCallback.getAliases();
+        if (aliases.size() != 1) {
+            throw new CentralRepoException("Alias add query failed");
+        }
+        
+        return aliases.iterator().next();
+    }
+    
+    /**
+     * Removes an alias from the specified Persona.
+     *
+     * @param persona Persona for which the alias is being removed.
+     * @param alias Alias to remove.
+     *
+     * @throws CentralRepoException If there is an error in removing the alias.
+     */
+    static void removePersonaAlias(Persona persona, PersonaAlias alias) throws CentralRepoException {
+        String deleteClause = " DELETE FROM persona_alias WHERE id = " + alias.getId();
```

Comment:
Same comment as before.

Since this is deleting the row with the row id of the alias - it doesn't need the Persona as argument.